jarviscli/plugins/voice_control.py

```diff
@@ -1,5 +1,7 @@
 import os
-from plugin import plugin, require
+from jarviscli.plugin import plugin, require
```

Comment:
Why this change? Importing jarviscli.plugin actually works for you?

Reply:
yes it does, and i was getting errors from vs code as well about not finding the file and couldnt get the suggestions. but yes, it works for me in this format